linkerd/app/src/dst/mod.rs

```diff
@@ -11,8 +12,9 @@ use tower_grpc::{generic::client::GrpcService, Body, BoxBody};
 pub struct Config {
     pub control: ControlConfig,
     pub context: String,
-    pub get_suffixes: Vec<dns::Suffix>,
-    pub profile_suffixes: Vec<dns::Suffix>,
+    pub get_suffixes: IndexSet<dns::Suffix>,
+    pub get_networks: IndexSet<ipnet::IpNet>,
+    pub profile_suffixes: IndexSet<dns::Suffix>,
```

Comment:
TIOLI: might be worth having doc comments explaining what all these similarly-named fields actually _do_?